pyanaconda/kickstart.py

```diff
@@ -929,9 +929,8 @@ def execute(self, storage, ksdata, instClass):
 
         cmd = "/usr/bin/firewall-offline-cmd"
         if not os.path.exists(util.getSysroot() + cmd):
-            if firewall_proxy.FirewallEnabled:
-                msg = _("%s is missing. Cannot setup firewall.") % (cmd,)
-                raise KickstartError(msg)
+            msg = _("%s is missing. Cannot setup firewall.") % (cmd,)
+            raise KickstartError(msg)
```

Comment:
This error will abort the installation, no? Do we really want to do this if the firewall mode is disabled or default?

Reply:
Good point! I should have looked better and noticed it is not just not a simple log message. Code blindness I guess. :)

So I'll just put the check back with correct API usage.